tasks/abideExtract.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
      src: [
        'public/dist/*.js',
      ],
-     dest: 'locale/templates/LC_MESSAGES/messages.pot',
+     dest: '../payments-l10n/locale/templates/LC_MESSAGES/payments-ui.pot',
```

Comment:
what does `../payments-l10n` point to? Does that need to be a separate source checkout? If so, the dev docs need updating: https://github.com/mozilla/payments-ui#developers